lec05/cw01/formValidation.js

```diff
@@ -0,0 +1,86 @@
+// TASK REQUIREMENTS
+// Login field validation must be without regex
+// Login must be a valid email
+// Password must be at least 6 character
+function validate() {
+  const data = extractData();
```

Comment:
change signature to `validate (data){}`
and call with `validate(extractData())`